source/_integrations/alarmdecoder.markdown

```diff
@@ -80,6 +80,11 @@ panel_display:
   required: false
   default: false
   type: boolean
+autobypass:
+  description: "If this is set to `true`, then when arming (home or away), it will automatically bypass all open zones (sending '6#')".
```

Comment:
```suggestion
  description: "If this is set to `true`, then when arming (home or away), it will automatically bypass all open zones (sending '6#')."
```